### DISCUSSION TITLE
急了急了，有人急了

### DISCUSSION THREAD

**Discussion by wu162** [vuepress-theme-hope/vuepress-theme-hope#3018] [Q&A] [answered]
我按照vuepress官方教程添加了

```
---
permalink: /hello-world
---
```

但是左边栏就没有了，然后增加sidebar: true ,页面直接404了。。。

**Comment by Mister-Hope** [comment]
谁教你填sidebar: true 的，你就去找谁。

**Reply by wu162** [reply]
> 谁教你填sidebar: true 的，你就去找谁。

后面那个是我自己尝试的，没有成功所以过来问，有必要这么戾气重吗？

**Reply by wu162** [reply]
> 谁教你填sidebar: true 的，你就去找谁。

哦，您的项目耶，惹不起，我不用了行吧

**Reply by Mister-Hope** [reply, marked as answer]
很高兴你放弃使用本项目。

对于开源项目，最重要的就是遵循文档（following docs)，我们的 [文档](https://theme-hope.vuejs.press/config/theme/layout.html#sidebar-config) 从未告诉你你可以在 sidebar 设置 true。

如果你喜欢尝试不一样的东西，欢迎使用具有技术支持的项目并付费购买服务获得技术支持，并通过技术支持获得帮助。开源项目作者并未从你身上获得任何收入，因此对于开源项目来说，仔细阅读文档应是用户的义务，并且用户应该在讨论和议题出尽可能详尽的列出自己遇到的问题（包括配置、操作与非预期结果）。

如果开源项目用户拒绝在议题和讨论区提供详细信息，或者他们坚持在不仔细阅读文档的情况下在议题或评论中要求他人为其寻找文档或解答，维护者拒绝回答此类用户的问题是合理的，因为项目维护者都有自己的家庭/事业/工作要处理。这对于只在GitHub提供了捐赠入口，而从未在文档或项目主页中展示地址并鼓励捐赠的本项目更为合理。

另：
1. 如果你觉得我让你向提议 sidebar: true 的人进行询问是一种冒犯，那你不读文档要求他人解答为何不是对他人以及项目维护者的一种冒犯。
1. 为了避免伤害你脆弱的心灵，你将被阻止与此组织交流，祝好。
